src/lib.rs

```diff
@@ -125,6 +167,11 @@ pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
     }
     println!("Collected {} stars", stars.len());
 
+    match remaining {
+        10 | 0...5 => println!("Warning: You have {} out of {} requests remaining. Your request limit will reset at {}", remaining, total, timestamp_str),
```

Comment:
As I mentioned in another comment, let's output this to `stderr` using [`eprintln!`](https://doc.rust-lang.org/std/macro.eprintln.html) to better communicate that there is a problem the user should fixcomment,